tests/e2e/chain/export.go

```diff
@@ -8,16 +8,21 @@ type ChainMeta struct {
 }
 
 type Validator struct {
-	Name          string `json:"name"`
-	ConfigDir     string `json:"configDir"`
-	Index         int    `json:"index"`
-	Mnemonic      string `json:"mnemonic"`
-	PublicAddress string `json:"publicAddress"`
+	Name           string `json:"name"`
+	ConfigDir      string `json:"configDir"`
+	Index          int    `json:"index"`
+	Mnemonic       string `json:"mnemonic"`
+	PublicAddress  string `json:"publicAddress"`
+	PublicAddress2 string `json:"publicAddress2"`
+	PublicKey      string `json:"publicKey"`
+	OperAddress    string `json:"operAddress"`
```

Comment:
Same as with `Chain`, we should minimize the changes here as much as possible for backward compatibility. Why do we need all these?

Reply:
Seems that we need a second address for the delegator to overwrite the vote, can we create it during the test itself? That way, we don't have to modify these structs to ensure backward compatibility and also wouldn't break the Validator's abstraction